invenio_drafts_resources/services/minter.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+
+"""Draft minter."""
+
+from invenio_pidrelations.contrib.versioning import PIDNodeVersioning
+from invenio_pidstore.minters import recid_minter_v2
+from invenio_pidstore.models import PersistentIdentifier, PIDStatus
+from invenio_pidstore.providers.recordid_v2 import RecordIdProviderV2
+
+
+# Keep `record_uuid` to comply with minter interface
+def conceptrecid_minter_v2(record_uuid=None, data=None):
+    """Mint the Concept RECID, reserves it for the record."""
+    conceptrecid = PersistentIdentifier.create(
+        pid_type='recid',
+        pid_value=str(RecordIdProviderV2.generate_id()),
+        status=PIDStatus.RESERVED,
+    )
+
+    data['conceptrecid'] = conceptrecid.pid_value
+    return conceptrecid
+
+
+# By default both recid and conceptrecid will be RESERVED
+RecordIdProviderV2.default_status_with_obj = PIDStatus.RESERVED
```

Comment:
Shelve: This is a global assignment that may have unintended consequences + makes it hard to test. We should revisit this eventually. Made issue: https://github.com/inveniosoftware/invenio-pidstore/issues/133

Reply:
I could not agree more. IMO this should be an instance attribute, not class. The main issue is that refactoring that would be a huge breaking change in `invenio-pidstore`, which is at the core of every Invenio repository... And IIRC, the idea for now is not to touch those, or keep to a minimum. 

I actually had to do a PR there to add this, but had to keep it back compat...

I think all this we can tackle once we have an RDM 1.0.0 out :) 🚀